Sources/xcproj/XcodeProj.swift

```diff
@@ -58,40 +58,131 @@ final public class XcodeProj {
 
 extension XcodeProj: Writable {
 
+    /// Writes project to the given path.
+    ///
+    /// - Parameter path: path to `.xcodeproj` file.
+    /// - Parameter override: if project should be overridden. Default is true.
+    ///   If false will throw error if project already exists at the given path.
     public func write(path: Path, override: Bool = true) throws {
         try path.mkpath()
         try writeWorkspace(path: path, override: override)
         try writePBXProj(path: path, override: override)
-        try writeSharedData(path: path, override: override)
+        try writeSchemes(path: path, override: override)
+        try writeBreakPoints(path: path, override: override)
+    }
+
+    /// Returns workspace file path relative to the given path.
+    ///
+    /// - Parameter path: `.xcodeproj` file path
+    /// - Returns: worspace file path relative to the given path.
+    public func workspacePath(_ path: Path) -> Path {
```

Comment:
I think this method can be static since it doesn't need anything from the `XcodeProj` instance.

Reply:
I'd add a simple unit test to make sure the path si properly generated.